source/patterns/01-molecules/info-bar/info-bar.twig

```diff
@@ -0,0 +1,15 @@
+{%- import 'atoms-html' as html -%}
```

Comment:
This file needs works so it works properly with specimen pages as well as individual pattern examples.